created elevator and wrist subsystems

AlenaNguyen opened the pull request.

AlenaNguyen committed:
created elevator and wrist subsystems

AngelaKimmy commented on src/main/java/frc/robot/RobotMap.java:
```diff
@@ -57,6 +57,15 @@ public static final class mapCollector {
     // public static final int PIVOT_ABSOLUTE_ENCODER_DIO = 1;
   }
 
+  public static final class mapElevator {
```
Remember to bump the motors by 10 for each subsystem

TaylerUva commented on src/main/java/frc/robot/subsystems/Elevator.java:
```diff
@@ -0,0 +1,81 @@
+    rightMotor.set(ControlMode.Position, position);
+  }
+
+  public double getElevatorEncoderCounts() {
+    return rightMotor.getSelectedSensorPosition();
+  }
```
We probably want to a getElevatorDistance that converts counts to inches (or meters)

TaylerUva replied:
for robot preferences and things we should use distance so it's easier to read later

TaylerUva commented on src/main/java/frc/robot/subsystems/Wrist.java:
```diff
@@ -0,0 +1,52 @@
+    wristMotor.set(ControlMode.PercentOutput, speed);
+  }
+
+  public void setWristPosition(double position) {
+    position = MathUtil.clamp(position, prefWrist.WristMinPos.getValue(),
+        prefWrist.WristMaxPos.getValue());
+
+    wristMotor.set(ControlMode.Position, position);
+  }
```
We probably want to name this setWristAngle

AlenaNguyen committed:
resolved comments

- changed motor IDs
- added feed forward values
- added getElevatorDistance function

AlenaNguyen committed:
removed redundant motor name
